io.github.amourspirit.LibrePythonista_PyEditor.yml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.amourspirit.LibrePythonista_PyEditor
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: cell_edit
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=home
+  - --filesystem=~/.var/app/org.libreoffice.LibreOffice:ro
+
+modules:
+  - name: cell_edit
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "librepythonista_python_editor==0.1.8" --no-build-isolation
+      - install -Dm755 cell_edit /app/bin/cell_edit
+      - install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -Dm644 $FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/metainfo/$FLATPAK_ID.metainfo.xml
+    sources:
+      - type: git
+        url: https://github.com/Amourspirit/LibrePythonista_PyEditor
+        tag: release-v0.1.1
+      - type: file
+        path: cell_edit
+      - type: file
+        path: internal/ui/res/icon.svg
+      - type: file
+        path: misc/io.github.amourspirit.LibrePythonista_PyEditor.metainfo.xml
```

Comment:
You can't list these files here, as they're not in this repo.

```suggestion
```

Reply:
@Amourspirit Just remove the files I asked you and apply this change here, and things should work.